snapshot/snapshot.go

```diff
@@ -509,7 +495,7 @@ func (o *snapshotter) Remove(ctx context.Context, key string) error {
 	}
 
 	// For example: remove snapshot with key sha256:c33c40022c8f333e7f199cd094bd56758bc479ceabf1e490bb75497bf47c2ebf
-	log.L.Debugf("[Remove] snapshot with key %s snapshot id %s", key, id)
+	log.L.Infof("[Remove] snapshot with key %s snapshot id %s", key, id)
```

Comment:
Please don't print it as INFO, it is tons of log lines

Reply:
Fixed